chaoscenter/web/src/views/Login/__tests__/LoginPage.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import LoginPageView from '../LoginPage';
+const mockHandleLogin = jest.fn();
+const mockGetString = jest.fn((key: string) => {
+  const strings = {
+    welcomeToLitmus: 'Welcome to Litmus',
+    LoginDescription: 'Please sign in to your account'
+  };
+  return strings[key as keyof typeof strings] || key;
+});
```

Comment:
Instead of mocking the strings, use `<StringContext.Provider>` which is also a part of `TestWrapper`, doing this will ensure you always assert the key. 

With your approach if the text value changes in future this test will still pass as the key: value pair is created within the test file itself. Use `<TestWrapper>` instead and assert the key for better stability.